data/card_catalogue/upgradeable-1.json

```diff
@@ -1,40 +1,52 @@
 [
   {
-    "title": "Whip",
-    "upgrade": "Pitch",
+    "title": "Toss",
+    "upgrade": "Throw",
     "tier": "blue",
     "details": {
-      "rules": "Throw an object up to 10 squares",
-      "flavour": "If you're going to throw in the towel, at least do it properly.",
-      "image": "assets/core/card/illustration/Whip.png"
+      "rules": {
+        "active": "Throw an object accurately at a target up to 5 metres away.",
```

Comment:
> as rooms aren't very large there wasn't much incentive to upgrade here. opting for accuracy instead of power

c25d01e12eeb8af33a633a7d3df8ca9bf1763863

@DuncanUszkay1 Thoughts on the change?